addons/medical_status/functions/fnc_setUnconsciousState.sqf

```diff
@@ -31,6 +31,9 @@ _unit setVariable [VAR_UNCON, _active, true];
 // Stop AI firing at unconscious units in most situations (global effect)
 [_unit, "setHidden", "ace_unconscious", _active] call EFUNC(common,statusEffect_set);
 
+// Block radio on unconsciousness for compatibility with captive module
+[_unit, "blockRadio", "ace_unconscious", _active] call EFUNC(common,statusEffect_set);
```

Comment:
This is the wrong place for this. See postInit of medical_feedback

Reply:
Non-player units also need to have radio disabled, though, in case of RC/switchUnit

Reply:
Oh true, I'm falsely equivalating two different things (the setVolume which belongs in feedback and this status effect which does belong here in status). This looks good, thanks for correcting me 👍